code/modules/mob/living/carbon/human/species.dm

```diff
@@ -805,6 +809,9 @@ GLOBAL_LIST_EMPTY(features_by_species)
 	if(slot in no_equip)
 		if(!I.species_exception || !is_type_in_list(src, I.species_exception))
 			return FALSE
+	if(!(I.bodytypes_whitelist & H.dna.species.bodytype))
+		to_chat(H, span_warning("Your species cannot wear this item!"))
```

Comment:
I'm noticing its also in this proc, but something named `can_*` should never have side effects.

I'll let this go because the other ones have it and a huge refactor would be non-trivial if you can check something for me--if I, a monkey, have an item I can't equip in my hand, and hover it over its slot (to see the red/green silhouette), does it spit a message in the chat?

Reply:
removed var/bodytypes_whitelist